### ISSUE TITLE
connection failed

### ISSUE THREAD

**Issue by AnjaliNirmale** [ricohapi/theta-client#80]
while connecting to the Theta camera through to iOS application.The user, who is on iOS version 17.4.1 is experiencing a timeout error with the following details:
Error Domain=NSURLErrorDomain Code=-1001 “The request timed out.” UserInfo={_kCFStreamErrorCodeKey=-2103, _NSURLErrorFailingURLSessionTaskErrorKey=LocalDataTask <3EC9BA23-7212-4579-85DD-00355DC28141>.<1>, _NSURLErrorRelatedURLSessionTaskErrorKey=(
“LocalDataTask <3EC9BA23-7212-4579-85DD-00355DC28141>.<1>”
), NSLocalizedDescription=The request timed out., NSErrorFailingURLStringKey=https://192.168.1.1/osc/info, NSErrorFailingURLKey=https://192.168.1.1/osc/info, _kCFStreamErrorDomainKey=4}

and camera Details :-
Firmware Version:2.51.0
Manufacturer: Ricoh Company, Ltd.
Model: RICOH THETA X

Interestingly,my camera works fine with the Theta app.
Thank you in advance for your assistance.

**Comment by simago** [comment]
@AnjaliNirmale Theta does not support HTTPS. 
Please try `http://192.168.1.1/osc/info`.

**Comment by AnjaliNirmale** [comment]
@simago   yes I agreed.. I m sending the http request, but I m unsure why the error response shows https..here is my request
Request: <URLRequest [
		URL: "http://192.168.1.1/osc/info",
		Method: "GET",
		Headers: [
			"Content-Length": "0"
		],
		CachePolicy: UseProtocolCachePolicy
	]

**Comment by AnjaliNirmale** [comment]
@simago any update.?

**Comment by simago** [comment]
@AnjaliNirmale `info.plist` of our [iOS demo application](https://github.com/ricohapi/theta-client/tree/main/demos/demo-ios) includes following descriptions:
```
        <key>NSAppTransportSecurity</key>
	<dict>
		<key>NSAllowsArbitraryLoads</key>
		<false/>
		<key>NSExceptionDomains</key>
		<dict>
			<key>192.168.1.1</key>
			<dict>
				<key>NSIncludesSubdomain</key>
				<false/>
				<key>NSTemporaryExceptionAllowsInsecureHTTPLoads</key>
				<true/>
				<key>NSTemporaryExceptionRequiresForwardSecrecy</key>
				<false/>
			</dict>
		</dict>
	</dict>
```
I hope this helps.
